plugins/modules/azure_rm_vmsku_info.py

```diff
@@ -0,0 +1,307 @@
+                                    description:
+                                        - An invariant if the feature is measured by quantity.
+                                    type: str
+                                    returned: always
+                                    sample: "True"
+        capabilities:
+            description:
+                - A name value pair to describe the capability.
+            returned: always
+            type: complex
+            contains:
+                name:
+                    description:
+                        - An invariant to describe the feature.
+                    type: str
+                    returned: always
+                    sample: "ultrassdavailable"
+                value:
+                    description:
+                        - An invariant if the feature is measured by quantity.
+                    type: str
+                    returned: always
+                    sample: "True"
+        restrictions:
+            description:
+                - The restrictions because of which SKU cannot be used. This is empty if there are no restrictions.
+            returned: always
+            type: complex
+            contains:
+                type:
+                    description:
+                        - The type of restrictions.
+                    type: str
+                    returned: always
+                    sample: "location"
+                values:
+                    description:
+                        - The value of restrictions. If the restriction type is set to location. This would be different locations where the SKU is restricted.
+                    type: str
+                    returned: always
+                    sample: ["eastus"]
+                restriction_info:
+                    description:
+                        - The information about the restriction where the SKU cannot be used.
+                    returned: always
+                    type: complex
+                    contains:
+                        locations:
+                            description:
+                                - Locations where the SKU is restricted.
+                            type: list
+                            sample: ["location"]
+                        zones:
+                            description:
+                                - List of availability zones where the SKU is restricted.
+                            type: list
+                            sample: ["1", "2"]
+                reason_code:
+                    description:
+                        - The reason for restriction.
+                    type: str
+                    sample: "QuotaId"
+'''
+
+try:
+    from azure.core.exceptions import HttpResponseError
+except ImportError:
+    # This is handled in azure_rm_common
+    pass
+
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase, azure_id_to_dict
```

Comment:
unused-import: Unused azure_id_to_dict imported 
```suggestion
from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase
```

Reply:
Removed, thanks

Reply:
Thanks for your feedback! Everything is OK, I will push for merged as soon as possible!